pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -2528,6 +2528,26 @@
         %rejected  ``noun+!>((~(has in rejected.cors-registry) u.origin))
       ==
     ::
+        [%scry @ @ ~]
+      =/  host  `i.t.tyl
+      =/  url  i.t.t.tyl
+      =/  server  (per-server-event [eny *duct now rof] server-state.ax)
+      =/  [=action suburl=@t]
+        (get-action-for-binding:server host url)
+      =/  invalid  (error-page 400 %.y url "invalid scry: {<action>} {<suburl>}")
+      :^  ~  ~  %noun
+      !>  ^-  octs
```

Comment:
At the very least you'll also want a `response-header` here. Might be best to just produce a `$>(%start http-event:http)` where `complete=&`, the runtimes already have logic for handling such nouns.